schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth date,
     escape_attempts integer,
     neutered boolean,
-    weight_kg decimal
+    weight_kg decimal,
+    species varchar(100)
 );
```

Comment:
- [ ] Instead of editing the previous animal schema directly, you should have just had another query to alter the animal table. Something like the example below.
`ALTER TABLE animals ADD species varchar(255);`